Mathlib/Data/ENNReal/Basic.lean

```diff
@@ -510,6 +510,16 @@ theorem toNNReal_nat (n : ℕ) : (n : ℝ≥0∞).toNNReal = n := by
 theorem toReal_nat (n : ℕ) : (n : ℝ≥0∞).toReal = n := by
   rw [← ENNReal.ofReal_natCast n, ENNReal.toReal_ofReal (Nat.cast_nonneg _)]
 
+lemma _root_.Nat.cast_toNNReal (N : ℕ) :
```

Comment:
I'm not sure about this name - maybe `ENNReal.toNNReal_natCast_eq_toNNReal`? (`ENNReal.toNNReal_natCast` is what `ENNReal.toNNReal_nat` above should be called.)

Reply:
I have renamed the lemmas accordingly, and also done the same for `ENNReal.to{Real,NNReal}_nat`

Reply:
I [asked on zulip](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/Naming.20convention/near/500597665), to be more certain. Ok, seems like a dependent PR to prepare...

Reply:
#22071 did this